.github/dependabot.yml

```diff
@@ -0,0 +1,39 @@
+# https://docs.github.com/en/code-security/dependabot
```

Comment:
Looking on the https://github.com/opendatahub-io/kubeflow/pulls?q=is%3Apr+author%3Aapp%2Fdependabot - are we sure we want this in here? Looks like we closed unmerged almost all of these PRs.

Reply:
This is different config than what was in effect at that time. This now only creates security PRs and it creates single PR for all security things it finds. So it's another sec alerting thing, in effect. Noise is minimal. There was nothing in opendatahub-io/main as far as I remember created by the bot recently. We'll have the same thing here.

Reply:
kicked out dependabot